.github/workflows/testing.yml

```diff
@@ -0,0 +1,74 @@
+name: testing
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.8'
+          - '2.9'
+
+    steps:
+      - name: Clone the connector
+        uses: actions/checkout@v2
+
+      - name: Setup tarantool ${{ matrix.tarantool }}
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Setup golang for the connector and tests
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+
+      - name: Setup node.js for tests
```

Comment:
Why do we need node.js for this tests?

Reply:
Hmm, I was sure that some tests don't work until I install `nodejs`. More accurately, I got a failure when starting a tarantool instance via `config.lua`. For some reason, I had such behavior locally. But it looks like it works without nodejs on GitHub runners.